main/src/main/java/org/mobilitydata/gtfsvalidator/validator/DuplicateRouteNameValidator.java

```diff
@@ -107,33 +108,39 @@ private boolean areRoutesFromSameAgency(
   }
```

Comment:
To make the code in the main validate() method more readable, I'd suggest passing the two `Route` objects into this `areRoutesFromSameAgency()` method and pulling out the agency IDs here.